ember-electron/utils/squirrel.js

```diff
@@ -0,0 +1,68 @@
+const { spawn } = require('child_process');
+const { app } = require('electron');
+const path = require('path');
+
+function handleSquirrelEvents() {
+    if (process.argv.length === 1) {
+        return false;
+    }
+
+    const appFolder = path.resolve(process.execPath, '..');
+    const rootAtomFolder = path.resolve(appFolder, '..');
+    const updateDotExe = path.resolve(path.join(rootAtomFolder, 'Update.exe'));
+    const exeName = path.basename(process.execPath);
+
+    const spawnUpdate = function(args) {
+        let spawnedProcess, error;
+
+        try {
+            spawnedProcess = spawn(updateDotExe, args, {detached: true});
+        } catch (error) {}
+
+        return spawnedProcess;
+    };
+
+    const createShortcuts = function(locations) {
+        return spawnUpdate(['--createShortcut', exeName, '-l', locations.join(',')]);
+    };
+
+    const removeShortcuts = function() {
+        return spawnUpdate(['--removeShortcut', exeName]);
+    };
+
+    const squirrelEvent = process.argv[1];
+    switch (squirrelEvent) {
+        case '--squirrel-install':
+        case '--squirrel-updated':
+        // Optionally do things such as:
```

Comment:
This reminds me to verify a proper coverage of ESLint. Could you please fix the indentation to match https://eslint.org/docs/rules/indent#switchcase (4 spaces instead of 2 in the example)? Thanks!